tests/playwright/sanity/includes/widgets/google-maps.test.ts

```diff
@@ -35,32 +35,3 @@ test( 'Verify Google maps controls', async ( { page, apiRequests }, testInfo ) =
 	currentHeight = await googleMapsWidget.getHeight( true );
 	expect( String( currentHeight ) ).toEqual( height );
 } );
-
-test( 'Check that Google Maps widget css file is loaded from `assets/css` when `Improved CSS Loading` experiment is active and SCRIPT_DEBUG is false', async ( { page, apiRequests }, testInfo ) => {
-	// Arrange.
-	const wpAdminPage = new WpAdminPage( page, testInfo, apiRequests );
-
-	await wpAdminPage.setExperiments( { e_optimized_css_loading: true } );
-
-	const editorPage = await wpAdminPage.openNewPage(),
-		container = await editorPage.addElement( { elType: 'container' }, 'document' );
-
-	// Add widgets.
-	await editorPage.addWidget( 'google_maps', container );
-	await editorPage.getPreviewFrame().locator( '.elementor-widget-google_maps' ).waitFor();
-
-	// Act
-	await editorPage.publishAndViewPage();
-
-	// When the `Improved CSS Loading` experiment is active, the Google Maps css file is loaded from `assets/css`.
-	const proFilePath = await page.evaluate( () => document.querySelector( '.elementor-widget-google_maps > .elementor-widget-container > link' ).getAttribute( 'href' ) ),
-		isProFilePath = proFilePath.includes( 'assets/css/widget' ),
```

Comment:
The test checks whether the styles are in the `<body>`, this is no longer the case.

Besides, google maps widget test should check for google maps functionality, not style loading functionality.